frontend/src/api/db/Productdb.js

```diff
@@ -0,0 +1,103 @@
+import kb1 from "../../assets/img/kb1.jpeg"
+import kb2 from "../../assets/img/kb2.png"
+import kb3 from "../../assets/img/kb3.jpg"
+import kb4 from "../../assets/img/kb4.jpg"
+import k1 from "../../assets/img/k1.jpg"
+import k2 from "../../assets/img/k2.jpg"
+import k3 from "../../assets/img/k3.jpg"
+import k4 from "../../assets/img/k4.jpg"
+import deskpad from "../../assets/img/deskpad.png"
+const Productdb = [
+  {
+    id:0,
```

Comment:
To work with Mockbee it's necessary for items to have a `_id`. So add that while putting this in db.

Reply:
thanks sudip , i will change it.